OWNERS

```diff
@@ -1,22 +1,11 @@
 # Reviewers can /lgtm /approve but not sufficient for auto-merge without an
 # approver
 reviewers:
-- Rajakavitha1
-- stewart-yu
-- xiangpengzhao
-- zhangxiaoyu-zidif
+- ClaudiaJKang
+- ianychoi
 
 # Approvers have all the ability of reviewers but their /approve makes
 # auto-merge happen if a /lgtm exists, or vice versa, or they can do both
 # No need for approvers to also be listed as reviewers
 approvers:
-- bradamant3
-- bradtopol
-- chenopis
-- kbarnard10
-- mistyhacks
-- ryanmcginnis
-- steveperry-53
-- tengqm
-- zacharysarah
-- zparnold
+- gochist
```

Comment:
@gochist 👋 FYI: approvers can't `/lgtm` or `/approve` their own PRs. I recommend having at least two approvers here so that you can also open PRs and get them approved. Is there someone else you can add?